.mise.toml

```diff
@@ -0,0 +1,180 @@
+[tools]
+python = "3.10"
+kubectl = "latest"
+uv = "latest"
+poetry = "latest"
+minikube = "latest"
+envsubst = "latest"
+yq = "latest"
+jq = "latest"
+ruff = "latest"
+direnv = "latest"
+
+[env]
+PLATFORM = "linux/amd64"
+# _.file = "etc/.env.development"
+
+[tasks."repo:setup"]
+description = "setup dependencies for the repo"
+run = "poetry install --with dev,test"
+
+
+[tasks."check:dead-code"]
+description = "check for dead python code"
+tools = ["poetry"]
+depends = ["repo:setup"]
+run = "poetry run vulture --min-confidence 80 ."
+
+[tasks."pkg:setup"]
+description = "setup virtualenv for pocketsizefund python package"
+run = """
+cd $root/pkg
+poetry install 
+"""
+
+[tasks."pkg:build"]
+description = "build the pocketsizefund python package"
+depends = ["pkg:setup"]
+run = """
+cd $root/pkg
+poetry build
+"""
+
+[tasks."pkg:publish"]
+description = "publish the pocketsizefund python package to PyPI registry"
+depends = ["pkg:build"]
+run = """
+cd $root/pkg
+poetry config pypi-token.pypi $PYPI_TOKEN
+poetry publish
+"""
+
+[tasks."pkg:test"]
+description = "unit tests for the pocketsizefund python package"
+depends = ["pkg:setup"]
+run = """
+cd $root/pkg
+poetry run coverage run --parallel-mode --module pytest
```

Comment:
I assume we don't need to point it at that specific `tests/` directory for this to work so that's fine if so.

Reply:
yea, the default for pytest is to have a `tests/` module alongside your package